### PR TITLE
Hs 346 add inspection screen load property data from database

### PR DESCRIPTION
**Pull request by dlvinh**

### CODE REVIEW

File: lib/features/booking/add_inspection_booking_screen.dart [hunk 1]
```diff
@@ -96,17 +99,26 @@ class AddInspectionBookingBody extends StatelessWidget {
...
+                            currency: state.price.currency,
+                            rentingPeriod: ' pw',
+                            onPressed: () {
+                              // TODO: implement BL
+                            },
+                          );
+                        }
+                        // TODO:handle when load property fail
+                        return const SizedBox(
+                          height: 100,
```
Comment (suesitran) [inline lib/features/booking/add_inspection_booking_screen.dart:120]: Do u need height here?

**Review by suesitran — COMMENTED** [review]
Just 1 comment

File: lib/features/booking/add_inspection_booking_screen.dart (hunk 1 above)
Comment (dlvinh) [reply lib/features/booking/add_inspection_booking_screen.dart:120]: @suesitran, we don't need it, I already removed it, please check the latest commit. Thanks

File: lib/features/booking/add_inspection_booking_screen.dart [hunk 2]
```diff
@@ -98,17 +101,24 @@ class AddInspectionBookingBody extends StatelessWidget {
...
+                      builder: (context, state) {
+                        if (state is PropertyDetailLoaded) {
+                          return BookedItemCard(
+                            propertyName: state.name,
+                            propertyType: state.type,
+                            propertyImage: state.photos.first,
+                            price: state.price.rentPrice,
+                            state: state.state,
+                            currency: state.price.currency,
+                            rentingPeriod: ' pw',
```
Comment (suesitran) [inline lib/features/booking/add_inspection_booking_screen.dart:114]: I think we have enum and string for this

**Review by suesitran — COMMENTED** [review]
Only 1 comment

File: lib/features/booking/add_inspection_booking_screen.dart (hunk 2 above)
Comment (dlvinh) [reply lib/features/booking/add_inspection_booking_screen.dart:114]: @suesitran I found out that in the UI does not required rentingPeriod, the `pw` value should be extracted from property's price. Also from our code, we did not have any logic to let user select that as well as store those `pw` value to firebase.
I will consider it as a bug and we should update it from UI and logic from AddProperty Flow

Comment (suesitran) [reply lib/features/booking/add_inspection_booking_screen.dart:114]: Please use localization here

Comment (dlvinh) [reply lib/features/booking/add_inspection_booking_screen.dart]: already apply localisation

File: lib/features/booking/add_inspection_success_booking_screen.dart [hunk 3]
```diff
@@ -71,7 +71,7 @@ class AddInspectionSuccessScreen extends StatelessWidget {
                   price: 4800,
                   state: 'Vitoria',
                   currency: Currency.aud,
-                  rentingPeriod: 'pw',
+                  paymentPeriod: HatSpaceStrings.current.pw,
```
Comment (suesitran) [inline lib/features/booking/add_inspection_success_booking_screen.dart:74]: Use "pm" here, as our price is always per month. I left a comment on figma

Comment (dlvinh) [reply lib/features/booking/add_inspection_success_booking_screen.dart]: already fixed it with 'pm'

**Review by suesitran — APPROVED** [review]

**Pull request merged**